viewer/packages/pointclouds/src/potree-three-loader/tree/PointCloudOctreePickerHelper.ts

```diff
@@ -205,6 +205,7 @@ export class PointCloudOctreePickerHelper {
     pickMaterial.minSize = nodeMaterial.minSize;
     pickMaterial.maxSize = nodeMaterial.maxSize;
     pickMaterial.classification = nodeMaterial.classification;
+    pickMaterial.objectAppearanceTexture = nodeMaterial.objectAppearanceTexture;
```

Comment:
Looks totally legit but found a funny bug. Texture contains not only hidden/visible information but also color, so because of that result of the picking is wrong when any of the objects has some color applied (this results in wrong `zoomToCursor` behaviour since picked point is not actually unde the cursor). See screenshot below with blue coloured pipe in `pointIndex` rendering mode that is used by picking. 

![Screenshot 2023-01-05 at 15 01 09](https://user-images.githubusercontent.com/45129444/210799639-4ece8534-7978-40ad-aee0-0b500705b40f.jpeg)

So, the solution would be to copy only "visibility" part of the texture and the rest of it should be with default values. Not sure where to place this logic, maybe in some utility function.

Reply:
Fixed by not using style color when rendering point index